shepherd/templates/stage_control.html

```diff
@@ -0,0 +1,384 @@
+<html>
```

Comment:
This looks very similar to `staff_gui.html`. There should be a better way to reuse this code.